### PR TITLE
get_experiment task

### PR DESCRIPTION
**Pull request by norlandrhagen**

### CODE REVIEW

File: cmip6_downscaling/methods/bcsd/flow.py [hunk 1]
```diff
@@ -20,6 +21,7 @@
     run_analyses,
 )
 
+dask.config.set({'temporary_directory': '/tmp/dask'})
```
Comment (jhamman) [inline cmip6_downscaling/methods/bcsd/flow.py:24]: We should set this in the runtime's `__init__` method.

File: cmip6_downscaling/methods/common/tasks.py [hunk 2]
```diff
@@ -55,29 +55,33 @@ def get_obs(run_parameters: RunParameters) -> UPath:
...
-    subset = subset_dataset(
-        ds, run_parameters.variable, run_parameters.train_period.time_slice, run_parameters.bbox
-    )
-    subset = subset.chunk({'time': 365})  # TODO: do better here
+    if time_subset == 'train_period':
+        time_period = run_parameters.train_period.time_slice
+    elif time_subset == 'predict_period':
+        time_period = run_parameters.predict_period.time_slice
+    else:
+        raise ValueError('time_subset arg is not train_period or predict_period.')
```
Comment (jhamman) [inline cmip6_downscaling/methods/common/tasks.py:81]:
```suggestion
time_period = getattr(run_parameters, time_subset)
```

I think will do the same thing.

File: cmip6_downscaling/methods/common/tasks.py [hunk 3]
```diff
@@ -55,29 +55,33 @@ def get_obs(run_parameters: RunParameters) -> UPath:
...
+        / time_subset
         / "{model}_{scenario}_{variable}_{latmin}_{latmax}_{lonmin}_{lonmax}_{train_dates[0]}_{train_dates[1]}_{predict_dates[0]}_{predict_dates[1]}".format(
             **asdict(run_parameters)
         )
     )
     if use_cache and (target / '.zmetadata').exists():
         print(f'found existing target: {target}')
         return target
-
+    print(target)
```
Comment (jhamman) [inline cmip6_downscaling/methods/common/tasks.py:71]:
```suggestion
```

File: cmip6_downscaling/methods/common/tasks.py [hunk 4]
```diff
@@ -55,29 +55,33 @@ def get_obs(run_parameters: RunParameters) -> UPath:
...
-def get_experiment(run_parameters: RunParameters) -> UPath:
-
-    # TODO: get train and predict data here
+def get_experiment(run_parameters: RunParameters, time_subset: str) -> UPath:
 
     target = (
         intermediate_dir
         / "get_experiment"
+        / time_subset
         / "{model}_{scenario}_{variable}_{latmin}_{latmax}_{lonmin}_{lonmax}_{train_dates[0]}_{train_dates[1]}_{predict_dates[0]}_{predict_dates[1]}".format(
```
Comment (jhamman) [inline cmip6_downscaling/methods/common/tasks.py:64]: We should update this to only include the dates in `time_period` (set below).

File: cmip6_downscaling/methods/common/tasks.py (hunk 2 above)
Comment (norlandrhagen) [reply cmip6_downscaling/methods/common/tasks.py:81]: Ah this is way cleaner!

**Pull request merged**